pkg/operator/controller/machineconfig/controller.go

```diff
@@ -37,6 +37,25 @@ import (
 //+kubebuilder:rbac:groups=machineconfiguration.openshift.io,resources=machineconfigpools,verbs=get;list;watch;create;delete
 //+kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch;patch
 
+// New returns a new MachineConfigReconciler instance.
+func New(mgr ctrl.Manager, impls ...impl) (*MachineConfigReconciler, error) {
```

Comment:
No error condition is handled, I think returning error is not required.

Reply:
> No error condition is handled, I think returning error is not required.

I have handled the error condition in the main.go